### DISCUSSION TITLE
Anybody Know How to Increase the scaling of the Splash screen?

### DISCUSSION THREAD

**Discussion by Romeo1984** [vicwomg/pikaraoke#383] [Q&A]
In past versions of PiKaraoke on pre-Bullseye, to get it to display properly on my Samsung 65" 4K television by performing:

`sudo raspi-config ->
7 Advanced Options -> A5 Resolution -> CEA Mode 4  1280x720 60Hz 16:9 -> OK -> Finish -> Would you like to reboot? <Yes>`

Now that we are on Bookworm and PiKaraoke is being launched into a browser window (Chromium for me) I can't figure out to get the Splash screen to not be so small. 

Here is a "Loading PiKaraoke" graphic I created and displaying with the command perfectly because it automatically scales the graphic:

`feh --bg-scale /home/pi/loading.png`

![IMG_6791](https://github.com/user-attachments/assets/f77131ab-808d-481d-9d33-e375cf452075)

But when the Splash page loads, it is very tiny:

![IMG_6792](https://github.com/user-attachments/assets/9d3aa3e6-8068-4f61-a1e3-8f85dd440a44)

What is the best way to scale this page? I was thinking of Chromium scaling switches, but I wanted to reach out to the community to see what the consensus is. 

Thanks!

**Comment by vicwomg** [comment]
Chromium uses the desktop resolution so you probably have to change it in
the desktop os. Here’s how my TV is set:

Pi icon in upper left > preferences > screen configuration > right click
screen > 1920x1080

Seems to be a reasonable size for me

**Reply by Romeo1984** [reply]
Being the FullPageOS image does not have a desktop to interact with, I opted for the following Chromium switch:
`--force-device-scale-factor=4.0 %U`

That got me the desired size splash screen. The videos and CDG files play normal size.
![IMG_6793](https://github.com/user-attachments/assets/9b338e65-e08a-4689-97df-d4ca71d3db6c)
